src/main/java/teammates/ui/webapi/action/SearchStudentsAction.java

```diff
@@ -28,6 +28,13 @@ public void checkSpecificAccessControl() {
         if (!userInfo.isInstructor && !userInfo.isAdmin) {
             throw new UnauthorizedAccessException("Instructor or Admin privilege is required to access this resource.");
         }
+        String entity = getNonNullRequestParamValue(Const.ParamsNames.ENTITY_TYPE);
+        if (!userInfo.isAdmin && "admin".equals(entity)) {
```

Comment:
Maybe extract the `admin` and 'instructor' string to a variable/const if possible?

```suggestion
        if (!userInfo.isAdmin && entity.equals("admin")) {
```

Reply:
Yeap sounds like a good idea, I’ll extract to a constant. The comparison order you suggested is not allowed by our lint settings, something about comparison of literals having to come first haha